tests/test_slots.py

```diff
@@ -269,9 +269,11 @@ def test_fsync_dir(self):
     def test_slots_advance_thread(self):
         with patch.object(MockCursor, 'execute', Mock(side_effect=psycopg.OperationalError)), \
                 patch.object(psycopg.OperationalError, 'diag') as mock_diag:
-            type(mock_diag).sqlstate = PropertyMock(return_value='58P01')
-            self.s.schedule_advance_slots({'foo': {'bar': 100}})
-            self.s._advance.sync_slots()
+            for err in ('58P01', '55000'):
+                type(mock_diag).sqlstate = PropertyMock(return_value=err)
+                self.s.schedule_advance_slots({'foo': {'bar': 100}})
+                self.s._advance.sync_slots()
+                self.assertCountEqual(self.s._advance._copy_slots, ["bar"])
```

Comment:
Shouldn't we just compare lists?
```suggestion
                self.assertEqual(self.s._advance._copy_slots, ["bar"])
```

Reply:
sure